app/views/user_mailer/added_to_collection_notification.text.erb

```diff
@@ -1,13 +1,13 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
+<%= t(".greeting", name: @user.default_pseud.byline) %>
 
-The collection maintainers of <%= @collection.title %> have added your work (<%= @work.title.html_safe %>) to their collection!
+<%= t(".part1", collection: @collection.title, work: @work.title.html_safe ) %>
 
-You are receiving this notification because you have previously elected to allow automatic inclusion of your works into collections on the Archive of Our Own.
+<%= t(".part2") %>
 
-If you would like to remove your work from this collection, please visit your "Approved Collection Items" page: <%= user_collection_items_url(@user) + "?approved=true" %>.
+<%= t(".text.part3", collection_items_url: user_collection_items_url(@user) + "?approved=true") %>
 
-If in future you would prefer to individually accept or reject requests to add your works to collections, please edit your Preferences (<%= "#{root_url}users/#{@user.default_pseud.name}/preferences" %>) and untick the box next to "Automatically agree to your work being collected by others in the Archive.".
+<%= t(".text.part4", link: user_preferences_url(@user)) %>
```

Comment:
Can we call this `preferences_url`? Something that'll make it clearer to Translation that it's a URL, not translatable text they need to worry about.

Reply:
Okay, I will also change any other instances of "link" to "something_url".